deployment/CONFIGURATION.md

```diff
@@ -42,7 +64,7 @@ cp hosts.yml.example hosts.yml
 | VALIDATOR_ADDRESS_PRIVATE_KEY: `"<private_key>"` | The private key for the specified validator address. |
 | syslog_server_port: `"<protocol>://<ip>:<port>"` | Optional port specification for bridge logs. This value will be provided by an administrator if required.  |
 
-`hosts.yml` can contain multiple hosts and bridge configurations (groups) at once.
+`hosts.yml` can contain multiple bridge configurations at once.
```

Comment:
The line 78 in the next section refers to `Configuration parameters` of `oracle/README.md` to get more info about values that can be applied as the deployment configuration parameters. 
First, the link is broken.
The second now `configuration parameters` are the mix of the oracle and UI parameters. Moreover some parameters will be applied to both components and some are component specific. I think the section below needs to be updated to bring more light which parameters should be configured for which component as well as which parameters could be skipped in the case of standalone deployment.

Reply:
Also, this section lacks the actual command to launch ansible :)
```
ansible-playbook -i hosts.yml site.yml
```

Reply:
As per discussion on a call with @akolotov , created https://github.com/poanetwork/tokenbridge/issues/127 to have more clear instructions and consistent parameters.

@phahulin The command is in the `EXECUTION.md` file.